### ISSUE TITLE
Ability to deploy Venia sample data instead of Luma sample data

### ISSUE THREAD

**Issue by drpayyne** [markshust/docker-magento#412]
It would be great if deploying Venia sample data was integrated into this project. Thanks!

Please refer: https://magento.github.io/pwa-studio/venia-pwa-concept/install-sample-data/

**Comment by markshust** [comment]
Good idea. I think something like a `bin/setup-pwa-studio-sampledata` script may work? Similar to how we have a `bin/setup-pwa-studio` script https://github.com/markshust/docker-magento/blob/master/compose/bin/setup-pwa-studio

**Comment by drpayyne** [comment]
@markshust, yep! sounds right.

**Comment by amitmaurya1024** [comment]
@markshust bin/setup-pwa-studio-sampledata needs this scipt. but this script is not there in bin folder. Have you developed this? If yes please share with us

**Comment by markshust** [comment]
I think it may be best to run PWA Studio code outside of Docker completely? I'm failing to see a need to run the Node.js app stuff through Docker. Please convince me otherwise 🤓

**Comment by drpayyne** [comment]
Certainly, yes! The PWA Studio app itself will run independently, _but_ the sample data Magento offers for Venia is different from the regular Luma sample data. This needs to be deployed to the Magento instance. More info can be found in link in the description.

**Comment by markshust** [comment]
@drpayyne thanks very much, I will check this out.
